applications/ContactStructuralMechanicsApplication/custom_processes/find_intersected_geometrical_objects_with_obb_for_contact_search_process.h

```diff
@@ -103,7 +103,13 @@ class KRATOS_API(CONTACT_STRUCTURAL_MECHANICS_APPLICATION) FindIntersectedGeomet
         ModelPart& rPart1,
         ModelPart& rPart2,
         const double BoundingBoxFactor = -1.0,
-        const Flags Options = BaseProcessType::INTERSECTING_CONDITIONS|BaseProcessType::INTERSECTING_ELEMENTS|BaseProcessType::INTERSECTED_CONDITIONS|BaseProcessType::INTERSECTED_ELEMENTS|BaseType::NOT_DEBUG_OBB|BaseType::SEPARATING_AXIS_THEOREM|BaseType::BUILD_OBB_FROM_BB
+        const Flags Options = BaseProcessType::INTERSECTING_CONDITIONS|
+            BaseProcessType::INTERSECTING_ELEMENTS|
+            BaseProcessType::INTERSECTED_CONDITIONS|
+            BaseProcessType::INTERSECTED_ELEMENTS|
+            ~BaseType::DEBUG_OBB|
```

Comment:
shouldn't this be sth with `AsFalse`?
```suggestion
            BaseType::DEBUG_OBB.AsFalse()|
```
Not sure though